### DISCUSSION TITLE
List to channel new products via API possible?

### DISCUSSION THREAD

**Discussion by castnettech** [Sellbrite/Sellbrite-API#37] [Q&A]
Hi, I’m nearing completion of my clients project that involves the sellbrite api. Is there a way to list to sales channels via the API, new or existing products? Or is it required to sign in to sellbrite and manually do this?

**Comment by rengler-godaddy** [comment]
Hello - Thanks for reaching out! 

There wouldn’t be a way to list to sales channels via the API. You would need to be signed into the Sellbrite dashboard to create listings.
